tests/unit/addon-tree-test-node.js

```diff
@@ -47,7 +49,7 @@ describe('Addon', function() {
       var addon = getMirageAddon({ configPath: 'tests/fixtures/config/environment-production-enabled' });
       var addonTree = addon.treeFor('addon');
 
-      expect(addonTree.inputTrees.length).to.be.equal(1);
+      expect(addonTree.inputTrees.length).to.not.equal(0);
     });
```

Comment:
The new version of `ember-cli-babel` made this input tree not exactly equal 1. It now exactly equals 2. 

Instead of changing this to be "2" and we're checking that it equals 0 in the case where mirage shouldn't be included, I just check that an inputTree structure exists.